src/main/resources/sql/maxwell_schema.sql

```diff
@@ -1,34 +1,58 @@
-CREATE DATABASE IF NOT EXISTS `maxwell`;
+SET @c = CONCAT('CREATE DATABASE IF NOT EXISTS ', @maxwell_db)
 
-CREATE TABLE IF NOT EXISTS `maxwell`.`schemas` (
+PREPARE maxwell_stmt from @c
+
+EXECUTE maxwell_stmt
+
+DEALLOCATE PREPARE maxwell_stmt
+
+SET @c = CONCAT('CREATE TABLE IF NOT EXISTS `', @maxwell_db, '`.`schemas` (
```

Comment:
I think it's ok.  We can quote the database name properly ( in backticks ), and `CREATE DATABASE` shouldn't be open to sql-injection in the same way that `INSERT INTO` or `SELECT * FROM` is.